sdk/src/main/scala/com/horizen/backup/BoxIterator.java

```diff
@@ -54,6 +56,24 @@ public Optional<BackupBox> nextBox() throws RuntimeException {
         return nextBox(false);
     }
 
+    public List<Box<Proposition>> getNextBoxes(int nElement, Optional<byte []> keyToSeek) {
+        if (keyToSeek.isPresent()) {
+            nElement += 1;
```

Comment:
Do we need to increment `nElement`, then add key to seek box to the resutl list and afterwords - remove that element?
Can we just do:
```
this.seekIterator(Utils.calculateKey(keyToSeek.get()).data());
this.nextBox();
```

Reply:
Yes this is much better!